package.json

```diff
@@ -24,8 +24,9 @@
   "dependencies": {
     "broccoli-funnel": "^2.0.1",
     "ember-cli-babel": "6.11.0",
-    "ember-lodash": "4.18.0",
+    "ember-cli-cjs-transform": "^1.3.0",
```

Comment:
Could you change `^` to `~`?

Reply:
@Exelord are we sticking with `ember-cli-cjs-transform` or going the vendor route? I pushed your recent feedback [here](https://github.com/Exelord/ember-rollbar-client/pull/35/commits/bac583c152e3af2f307d300877617863fea2f0bc), but if you want to go the vendor route, I'll need some more guidance there.

I'm not as familiar with broccoli trees; I follow bringing in the dependency through the vendor tree, but it's unclear how to import it in the service.